packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -309,7 +308,7 @@
 .usa-alert--success, 
 .usa-alert--error {
   .usa-alert__body {
-    padding-left: rem-override(2.9166666667rem);
+    padding-left: rem-override(2.91667rem);
```

Comment:
I think when we updated the version of USWDS, this rem value changed so I'm updating it here.